test/helper.rb

```diff
@@ -1,28 +1,13 @@
-require 'rubygems'
-require 'bundler'
+$LOAD_PATH.unshift(File.expand_path("../../", __FILE__))
+require "test-unit"
+require "fluent/test"
+require "fluent/test/helpers"
 
-require 'test/unit'
-require 'test/unit/rr'
+Test::Unit::TestCase.include(Fluent::Test::Helpers)
+Test::Unit::TestCase.extend(Fluent::Test::Helpers)
```

Comment:
Do we need this?

Reply:
We don't need this and we can remove this.

I made this change for the same reason as `$LOAD_PATH`.

Is it preferable to remove unnecessary codes rather than keep them aligned with the template?
If so, I will remove these lines.
(I feel that both have advantages and disadvantages, so I cannot judge.)

Reply:
https://github.com/fluent/fluentd/commit/a5fe470c201dd0fa60b74c5de407bf3bbf96f0e0 said that it's for `config_element`. I think that this is not a good design because unrelated methods such as `assert_equal_event_time`.

I think that we should improve this API in Fluentd itself.

So I think that we don't need this line in this plugin.

Reply:
Thanks! I see!
I will remove these lines!

Reply:
Fixed at https://github.com/fluent-plugins-nursery/fluent-plugin-flowcounter-simple/compare/55b4cf4093e976a26648c869f4060012e6f9a210..515d2d6fec584370fe39e9a4cebeba68293903b6